data_summaries/Lumos Grounding.json

```diff
@@ -0,0 +1,200 @@
+{
+    "lumos-grounding-complex_qa": {
```

Comment:
Thanks for adding all the extra details here. I will have @jadkabbara review this. It may take a bit of time, as we may need to hire someone to go and check the applicability of github/HF licenses to the data.

Reply:
Thanks! Let me know if there's anything needed